dnf+

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
- Optional change: wrong filename: `dnf` is expected.
- Common page flaws are not listed for other pages.